github.md

```diff
@@ -22,3 +32,73 @@ you should do:
+* optional: Link(s) to view changes, associated JIRA tickets, or other links that may be useful for understanding the changes in the PR.
+* optional: mention any person who may need to be aware of the changes in the PR (PM, UX, BO, or other devs)
+
+You should then assign one or more reviewers. (If you have more than one, you may want to mention in the description what each should be reviewing).
+
+Other notes: 
+"WIP" indicates a PR that is a work-in-progress that is not ready for review, but can be useful for comparing diffs as you work, or sharing in-progress work with others to see or comment on. In general, WIP PRs can be ignored.
+
+## Github and JIRA
+
+In general, if you are creating code on GitHub, commenting and review should happen there, with the code. If you are also using JIRA, you should add a comment with a link to the PR to the story in JIRA and move the story to the Review column to indicate that the PR is under review.
+
+(If changes are not commited to GitHub - like with Wordpress configurations - the comments and review should happen in JIRA; Mention reviewers by name in a comment on the ticket, what they should be reviewing, where they can view the changes, and then move the story to the Review column.)
+
+- - -
+
+## GitHub basics
+
+This process assumes you have: a terminal program (like iTerm with oh-my-zsh), an editor program (like Sublime Text or Atom), a github account and access to the MIT Libraries repos, and have already cloned the repos to your local machine. (For help with those steps, contact one of the MITLib devs.)
+
+*Start fresh*: Checkout master branch
+```git checkout master```
+
+*Get latest*: Get the latest changes that have been merged 
+```
+git fetch
+git merge
+```
+
+*Create a branch*: Create a new branch starting with master (or the branch you want to base your work on): This is the clean separate space where you will put your work.
+```git checkout -b [branchname]```
+
+*Do stuff*: open the repo in your editor and write code or make edits that relate to the fix/feature/experiment you are working on.
+
+*Commit your changes regularly*: once you have a few changes that add up to something (eg. 'added new js function to load rss' or 'added new css to support adjusted mini-cal alignment') or if you have to stop for a bit or switch to something else, add the files you are ready to commit, then commit them to your branch and push them to github.
+
+```
+git add [filename] 
+git add [filename] 
+...
+git commit
+[add a description of what's in the commit]
+git push
+```
+
+*Do more stuff*: keep working and committing (repeat steps 4 and 5) until your work for that fix/feature/experiment is as complete as you can make it.
+
+*Make a PR*: go to github and create a new PR for your branch against master/prod branch (see Pull Request Reviews above). 
+
+*Respond to PR feedback*: as feedback comes in, create additional commits to address and push them up.
```

Comment:
Maybe note to not squash or rebase at this point so reviewers can pick up where they left off? Squashing after the initial PR probably to be explicitly called out as a bad (until 👍 )